### PR TITLE
AddNewCopyWay

### PR DESCRIPTION
**Pull request by SharonZhang1**

### CODE REVIEW

**Review by JeneZhang — APPROVED** [review]

**Comment by SharonZhang1** [comment]
#sign-off

**Comment by JeneZhang** [comment]
#sign-off

File: docs/data-engineering/how-to-use-notebook.md [hunk 1]
```diff
@@ -119,6 +119,26 @@ When you open a notebook, you enter the co-editing mode by default, and every no
 
 :::image type="content" source="media\how-to-use-notebook\collaboration.png" alt-text="Screenshot showing a code cell with another user editing.":::
 
+### Share a notebook
+
+Sharing a notebook is a convenient way for you to collaborate with team members. Authorized workspace roles can view or edit/run notebook by default. Besides, we support share a notebook with granting specified permissions.
```
Comment (Court72) [inline docs/data-engineering/how-to-use-notebook.md:124]:
```suggestion
Sharing a notebook is a convenient way for you to collaborate with team members. Authorized workspace roles can view or edit/run notebooks by default. You can share a notebook with specified permissions granted.
```

File: docs/data-engineering/how-to-use-notebook.md [hunk 2]
```diff
@@ -119,6 +119,26 @@ When you open a notebook, you enter the co-editing mode by default, and every no
...
+
+1. Select the **Share** button on the notebook toolbar.
+
+   :::image type="content" source="media\how-to-use-notebook\open-sharenotebook-popup.png" alt-text="Screenshot showing where to select Share.":::
+
+1. Select corresponding category of **people who can view this notebook**. You could check **Share**, **Edit**, or **Run** to grant the permissions to the recipients.
```
Comment (Court72) [inline docs/data-engineering/how-to-use-notebook.md:130]:
```suggestion
1. Select the corresponding category of **people who can view this notebook**. You can check **Share**, **Edit**, or **Run** to grant the permissions to the recipients.
```

File: docs/data-engineering/how-to-use-notebook.md [hunk 3]
```diff
@@ -119,6 +119,26 @@ When you open a notebook, you enter the co-editing mode by default, and every no
...
+
+   :::image type="content" source="media\how-to-use-notebook\select-permissions.png" alt-text="Screenshot showing where to select permissions.":::
+
+1. After "Apply" the selection, you could either send directly or copy link to others, then the recipients can open the notebook with corresponding view by the permission.
+
+   :::image type="content" source="media\how-to-use-notebook\create-and-send-link.png" alt-text="Screenshot showing where to create and send link.":::
+
+1. To further manage your notebook permissions, you can find the “Manage permissions” entry in the *Workspace item list -> More options* to update the existing notebook access and permission.
```
Comment (Court72) [inline docs/data-engineering/how-to-use-notebook.md:138]:
```suggestion
1. To further manage your notebook permissions, you can find the "Manage permissions" entry in the **Workspace item list** > **More options** to update the existing notebook access and permission.
```

File: docs/data-engineering/microsoft-spark-utilities.md [hunk 4]
```diff
@@ -98,6 +98,14 @@ Copies a file or directory. Supports copy across file systems.
 mssparkutils.fs.cp('source file or directory', 'destination file or directory', True)# Set the third parameter as True to copy all files and directories recursively
 ```
 
+### Performant Copy file
```
Comment (Court72) [inline docs/data-engineering/microsoft-spark-utilities.md:101]:
```suggestion
### Performant copy file
```

File: docs/data-engineering/microsoft-spark-utilities.md [hunk 5]
```diff
@@ -98,6 +98,14 @@ Copies a file or directory. Supports copy across file systems.
...
+
+Provide a faster way of copying or moving files, especially large volume of data.
```
Comment (Court72) [inline docs/data-engineering/microsoft-spark-utilities.md:103]:
```suggestion
Provide a faster way of copying or moving files, especially large volumes of data.
```

**Comment by Court72** [comment]
@sharonzhang1, @JeneZhang,

Can you apply my suggestions to address some grammar/style issues? Also, can you update the following file name:
- docs/data-engineering/media/how-to-use-notebook/open-sharenotebook-popup.png
   - Add a hyphen between "share" and "notebook"

Thanks
#hold-off

**Comment by Court72** [comment]
#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by SharonZhang1** [comment]
#sign-off

**Comment by JeneZhang** [comment]
#sign-off

**Review by JeneZhang — APPROVED** [review]

**Pull request merged**